06.bowling_object/spec/game_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require_relative '../lib/game'
+
+describe Game do
+  describe 'play and score' do
+    subject do
+      game = Game.new
+      game.play(marks)
+    end
+
+    context 'when marks has strike and spare and no mark' do
```

Comment:
あ、 https://bootcamp.fjord.jp/practices/210 で要求されているテストパターンはyuzuuuさんが考えたテストパターンじゃないので説明文を考えるのが難しいですよね。

なので（特例として）ここはこんな感じでも良いと思います。

```suggestion
    context 'when requested pattern 1' do
```